.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull Request
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node_version: [10.x, 12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node_version }}
+        uses: actions/setup-node@v1
+        with:
+          node_version: ${{ matrix.node_version }}
+
+      - run: npm install
+
+      - run: npm test
+        env:
+          CI: true
```

Comment:
Do we perhaps want to run `npm run lint` too? (not sure whether this repo has it)

Reply:
`test` runs `lint` (which I generally think we should be doing):
```json
"test": "npm run lint && jest --config .jestrc.json"
```

Reply:
Ah! 👍